CHANGELOG.md

```diff
@@ -27,7 +27,9 @@ release.
 </tr>
 <tr>
     <td valign="top">
-<b><a href="doc/changelogs/CHANGELOG_V8.md#8.3.0">8.3.0</a></b><br/>
+<b><a href="doc/changelogs/CHANGELOG_V8.md#8.5.0">8.5.0</a></b><br/>
+<a href="doc/changelogs/CHANGELOG_V8.md#8.4.0">8.4.0</a><br/>
+<a href="doc/changelogs/CHANGELOG_V8.md#8.3.0">8.3.0</a><br/>
```

Comment:
Should we be replicating this? Not important for this merge, of course, just wondering.

Reply:
We have our own changelog in `deps/chakrashim`, although I haven't regenerated that yet. I think I'll need to do some additional work to get that in a good state, since right now I haven't cherry picked the commits across, just the changes.